doctest/extensions/mpi_reporter.h

```diff
@@ -196,6 +212,9 @@ struct MpiConsoleReporter : public ConsoleReporter {
           s << "\n";
         }
       }
+
+      MPI_Waitall(int(requests.size()), requests.data(), MPI_STATUSES_IGNORE);
```

Comment:
```suggestion
      MPI_Waitall(static_cast<int>(requests.size()), requests.data(), MPI_STATUSES_IGNORE);
```

is the cast even needed?

Reply:
yes you don't need it, it will be converted anyway